sycl/doc/extensions/experimental/sycl_ext_oneapi_non_uniform_groups.asciidoc

```diff
@@ -423,225 +424,102 @@ if (will_branch)
-----
+=== Chunk
 
-
-=== Tangle-Groups
-
-A tangle-group is a non-contiguous subset of a group representing work-items
-executing in a tangle. A tangle-group can therefore be used to capture all
-work-items currently executing the same control flow.
+A `chunk` represents a contiguous collection of work-items created by
+subdividing a group into equally sized parts, such that each work-item is a
+member of exactly one partition.
+The size of a chunk is a static (compile-time) property.
 
 
 ==== Creation
 
-New tangle-groups are created by partitioning an existing group, using the
-`get_tangle_group` free-function.
-
-NOTE: Creating a tangle-group may implicitly synchronize members of the
-`tangle_group` on some devices, since it may be necessary to wait for
-work-items to reconverge. For consistency, this synchronization is required by
-all implementations.
+A new `chunk` can only be created by partitioning an existing group with the
+`chunked_partition` function.
 
 [source, c++]
 ----
 namespace ext::oneapi::experimental {
 
-template <typename Group>
-tangle_group get_tangle_group(Group group);
+template <size_t ChunkSize, typename ParentGroup>
+chunk<ChunkSize, Group> chunked_partition(ParentGroup parent);
 
 } // namespace ext::oneapi::experimental
 ----
 
-_Constraints_: Available only if `sycl::is_group_v<std::decay_t<Group>> &&
-std::is_same_v<Group, sycl::sub_group>` is true.
+_Constraints_: Available only if `sycl::is_group_v<std::decay_t<ParentGroup>>`
+is true and `ParentGroup` is a `sycl::sub_group` or `sycl::chunk`.
+`ChunkSize` must be positive and a power of 2.
+
+_Preconditions_: `ChunkSize` must be less than or equal to the result of
+`parent.get_max_local_range()`.
+`parent.get_local_linear_range()` must be evenly divisible by `ChunkSize`.
 
-_Effects_: Synchronizes all work-items in the resulting `tangle_group`.
+_Effects:_ Blocks until all work-items in `parent` have reached this
+synchronization point.
 
-_Returns_: A `tangle_group` consisting of the work-items in `group` which are
-part of the same tangle.
+_Synchronization:_ The call in each work-item happens before any work-item
+blocking on the same synchronization point is unblocked.
+Synchronization operations used by an implementation must respect the memory
+scope reported by `ParentGroup::fence_scope`.
 
+_Returns_: A `chunk` consisting of all work-items in `parent` that are in the
+same partition as the calling work-item.
 
-==== `tangle_group` Class
+
+==== `chunk` Class
 
 [source, c++]
 ----
 namespace sycl::ext::oneapi::experimental {
 
-template <typename ParentGroup>
-class tangle_group {
+template <std::size_t ChunkSize, typename ParentGroup>
+class chunk {
 public:
   using id_type = id<1>;
   using range_type = range<1>;
   using linear_id_type = uint32_t;
   static constexpr int dimensions = 1;
   static constexpr sycl::memory_scope fence_scope = ParentGroup::fence_scope;
-    
+
+  operator fragment<ParentGroup>() const;
```

Comment:
`ParentGroup` could be `chunk` here, but `fragment`'s description does not allow to use `chunk` as a parent group of `fragment.

Reply:
Good catch!  It should be valid to call `binary_partition` with a `chunk`, so I've fixed that in 3e1b00152a.